pkg/clients/pagerduty.go

```diff
@@ -26,15 +26,15 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
-	"github.com/sapcc/go-pagerduty"
+	"github.com/PagerDuty/go-pagerduty"
```

Comment:
github.com/sapcc/go-pagerduty

Reply:
nope - I kicked it out, because of changes in the object and there is no benefit of the fork anymore - the origin covers needed functionality

Reply:
https://github.com/sapcc/go-pagerduty/commit/304f7ea6b5b6d37787f9fa87bfaa6f4a417c4346 is still not upstream